packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection.tsx

```diff
@@ -0,0 +1,14 @@
+const IndraAboutMeSection = () => {
+  return (
+    <section className="container mx-auto space-y-2">
```

Comment:
You could include a picture of yourself or an avatar to add a more personal and friendly touch.

Reply:
I actually don't have a lot of photo 😆 grabbing some recent one here. Was planning to get some user/profile NFT but haven't got a chance to look for something I'd like.

fixed at [`95b455c` (#16)](https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/16/commits/95b455c9eb185d7add941ff67d7bd8e182da0cb9)